textproto output fails for Get RPC

```
❯ gnmiClient --insecure get --path /state/system/version --format textproto
panic: reflect.Value.Interface: cannot return value obtained from unexported field or method

goroutine 19 [running]:
reflect.valueInterface(0x16ea640, 0xc00020afc0, 0x36, 0x16ea601, 0xc00020afc0, 0x36)
	/usr/local/go/src/reflect/value.go:1014 +0x1bc
reflect.Value.Interface(...)
	/usr/local/go/src/reflect/value.go:1003
github.com/gogo/protobuf/proto.(*TextMarshaler).writeAny(0x1e6b97c, 0xc0001e1f40, 0x17bc740, 0xc00020afc0, 0x1b9, 0xc0001c1600, 0x19569c0, 0x17bc740)
	/home/karim/go/pkg/mod/github.com/gogo/protobuf@v1.2.1/proto/text.go:592 +0x1ec
github.com/gogo/protobuf/proto.(*TextMarshaler).writeStruct(0x1e6b97c, 0xc0001e1f40, 0x17d4a80, 0xc00020afc0, 0x199, 0x199, 0x21006d0)
	/home/karim/go/pkg/mod/github.com/gogo/protobuf@v1.2.1/proto/text.go:453 +0x94a
github.com/gogo/protobuf/proto.(*TextMarshaler).Marshal(0x1e6b97c, 0x1934460, 0xc000304000, 0x1942380, 0xc00020afc0, 0x9, 0x0)
	/home/karim/go/pkg/mod/github.com/gogo/protobuf@v1.2.1/proto/text.go:894 +0x295
github.com/gogo/protobuf/proto.(*TextMarshaler).Text(0x1e6b97c, 0x1942380, 0xc00020afc0, 0x7ffeefbff994, 0x9)
	/home/karim/go/pkg/mod/github.com/gogo/protobuf@v1.2.1/proto/text.go:906 +0x6e
github.com/gogo/protobuf/proto.MarshalTextString(...)
	/home/karim/go/pkg/mod/github.com/gogo/protobuf@v1.2.1/proto/text.go:922
github.com/karimra/gnmiClient/cmd.printGetResponse(0x0, 0x0, 0xc00020afc0)
	/home/karim/github.com/karimra/gnmiClient/cmd/get.go:181 +0x9dd
github.com/karimra/gnmiClient/cmd.glob..func2.1(0xc0001dca40, 0xc0001e8680, 0xc0001e86c0, 0xc0001e2400, 0x0, 0x0, 0x0, 0xc0001e8670, 0x1, 0x1, ...)
	/home/karim/github.com/karimra/gnmiClient/cmd/get.go:156 +0x4ba
created by github.com/karimra/gnmiClient/cmd.glob..func2
	/home/karim/github.com/karimra/gnmiClient/cmd/get.go:97 +0x538
```

--- a/cmd/get.go
+++ b/cmd/get.go
@@ -23,7 +23,7 @@ import (
 	"sync"
 	"time"
 
-	"github.com/gogo/protobuf/proto"
+	"github.com/golang/protobuf/proto"
 	"github.com/google/gnxi/utils/xpath"
 	"github.com/openconfig/gnmi/proto/gnmi"
 	"github.com/spf13/cobra"
@@ -178,7 +178,7 @@ func init() {
 
 func printGetResponse(printPrefix string, response *gnmi.GetResponse) {
 	if viper.GetString("format") == "textproto" {
-		fmt.Printf("%s\n", indent(printPrefix, proto.MarshalTextString(response)))
+		fmt.Printf("%s\n", proto.MarshalTextString(response))
 		return
 	}
 	for _, notif := range response.Notification {

--- a/cmd/get.go
+++ b/cmd/get.go
@@ -23,12 +23,12 @@ import (
 	"sync"
 	"time"
 
-	"github.com/golang/protobuf/proto"
 	"github.com/google/gnxi/utils/xpath"
 	"github.com/openconfig/gnmi/proto/gnmi"
 	"github.com/spf13/cobra"
 	"github.com/spf13/viper"
 	"google.golang.org/grpc/metadata"
+	"google.golang.org/protobuf/encoding/prototext"
 )
 
 // getCmd represents the get command
@@ -178,7 +178,7 @@ func init() {
 
 func printGetResponse(printPrefix string, response *gnmi.GetResponse) {
 	if viper.GetString("format") == "textproto" {
-		fmt.Printf("%s\n", proto.MarshalTextString(response))
+		fmt.Printf("%s\n", indent(printPrefix, prototext.Format(response)))
 		return
 	}
 	for _, notif := range response.Notification {

--- a/cmd/subscribe.go
+++ b/cmd/subscribe.go
@@ -24,12 +24,12 @@ import (
 	"sync"
 	"time"
 
-	"github.com/golang/protobuf/proto"
 	"github.com/google/gnxi/utils/xpath"
 	"github.com/openconfig/gnmi/proto/gnmi"
 	"github.com/spf13/cobra"
 	"github.com/spf13/viper"
 	"google.golang.org/grpc/metadata"
+	"google.golang.org/protobuf/encoding/prototext"
 )
 
 type msg struct {
@@ -317,7 +317,7 @@ func printSubscribeResponse(meta map[string]interface{}, subResp *gnmi.Subscribe
 	switch resp := subResp.Response.(type) {
 	case *gnmi.SubscribeResponse_Update:
 		if viper.GetString("format") == "textproto" {
-			fmt.Printf("%s\n", proto.MarshalTextString(subResp))
+			fmt.Printf("%s\n", prototext.Format(subResp))
 			return
 		}
 		msg := new(msg)

--- a/go.mod
+++ b/go.mod
@@ -15,5 +15,6 @@ require (
 	github.com/spf13/viper v1.6.2
 	golang.org/x/crypto v0.0.0-20200302210943-78000ba7a073
 	google.golang.org/grpc v1.29.1
+	google.golang.org/protobuf v1.23.0
 	gopkg.in/yaml.v2 v2.2.4
 )

